src/extensions/removable.rs

```diff
@@ -1,193 +1,40 @@
-use crate::{RelmSetChildExt, RelmWidgetExt};
+use crate::RelmSetChildExt;
 use gtk::prelude::*;
 
 /// Widget types which can have widgets removed from them.
-pub trait RelmRemovableExt {
-    /// Type of children of the container.
-    type Child;
+pub trait RelmRemovableExt<W> {
```

Comment:
Is there a specific reason to use a generic parameter `W` instead of an associated type `Child`? I can image some cases where this might require some ugly type annotations because Rust can't know the generic type parameter.

Reply:
No, there isn't. I'll change it then.

Reply:
There might be a name conflict though if `RelmRemovableExt` and `RelmIterChildrenExt` are in the same scope.

Reply:
That's true. Probably, all traits that have `RelmIterChildrenExt` also implement `RelmRemovableExt`, or am I wrong. I think you could use `trait RelmIterChildrenExt: RelmRemovableExt { ... }` to use the `Child` type from one trait in the other trait, too. Or you just rename one to `RemoveChild` or `IterChild`.